SignalServiceKit/src/Messages/OWSDisappearingMessagesJob.m

```diff
@@ -266,12 +266,14 @@ - (void)becomeConsistentWithConfigurationForMessage:(TSMessage *)message
         TSIncomingMessage *incomingMessage = (TSIncomingMessage *)message;
         NSString *contactName = [contactsManager displayNameForPhoneIdentifier:incomingMessage.authorId];
 
-        [[[OWSDisappearingConfigurationUpdateInfoMessage alloc] initWithTimestamp:message.timestamp
+        // We want the info message to appear after the message.
+        [[[OWSDisappearingConfigurationUpdateInfoMessage alloc] initWithTimestamp:message.timestamp + 1
```

Comment:
TIOLI: I think it makes more sense for the info message to come before the text

e.g. if Alice:
- sends "Foo" 
- goes into settings and enables Disappearing messages
- sends "Bar"
- sends "Baz"

Bob will see:

- "Foo"
- "Alice enabled Disappearing Messages"
- "Bar (timer)"
- "Baz (timer)

To me it seems  a little weird that we'd build a scenario where Bob sees:

- "Foo"
- "Bar (timer)"
- "Alice enabled Disappearing Messages"
- "Baz (timer)

Reply:
Good point! Done.